File: game-db/src/main/java/com/jacey/game/db/entity/GmUserEntity.java
package com.jacey.game.db.entity;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

/**
 * @Description: GM账户
 * @Author: JaceyRuan
 * @Email: jacey.ruan@outlook.com
 */
@Data
@Document(collection = "gm_user")
public class GmUserEntity {

    @Id
    private int userId;

    /** 玩家名称 */
    private String username;

    /** MD5加密密码 */
    private String passwordMD5;

}

File: game-db/src/main/kotlin/com/jacey/game/db/entity/GmUserEntity.kt
package com.jacey.game.db.entity

import com.jacey.game.common.framework.mongo.DbDocument

/**
 * GM账户（Mongo 集合 gm_user）
 */
data class GmUserEntity(
    override val _id: Int,
    var username: String? = null,
    var passwordMD5: String? = null,
) : DbDocument<Int> {
    val userId: Int get() = _id

    companion object {
        const val COLLECTION_NAME = "gm_user"
    }
}